src/components/Dashboard/Alert/fixtures/AlertItem.fixture.js

```diff
@@ -40,6 +40,30 @@ export const unknownTypeAlert = {
   },
 };
 
+export const cephDiskInaccessibleAlert = {
```

Comment:
I suggest to move this ceph-specific alerts to StorageOverview `Alerts.fixture.js`

Reply:
ack!